index.html

```diff
@@ -5,11 +5,94 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="../src/styles.css" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
+    <header>
+        <nav class="header-nav">
+          <img src="../images/sumo1.jpg" alt="sumo" class="logo" />
+          <h2>sumo bakery</h2>
+          <input type="checkbox" id="check" />
+          <label for="check" class="check-btn">
+            <i class="fa fa-bars"><p class="menubtn-text">menu</p></i>
+          </label>
+          <ul class="nav-bar">
+            <li><a href="">Home </a></li>
+            <li><a href="">order </a></li>
+            <li><a href="">service </a></li>
+            <li><a href="">cake </a></li>
+            <li><a href="">feedback </a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+          
+        
```

Comment:
remember to look at your formatting when committing your changes for improved readability of the code - right-click on your editor in VSCode and choose "Format Document" and it will do all the work for you.